extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/discovery/HueLightDiscoveryService.java

```diff
@@ -47,6 +54,7 @@
  *         added representationProperty to discovery result
  * @author Thomas Höfer - Added representation
  * @author Denis Dudnik - switched to internally integrated source of Jue library
+ * @author Samuel Leisering - Added sensor support
  */
 @NonNullByDefault
 public class HueLightDiscoveryService extends AbstractDiscoveryService implements LightStatusListener {
```

Comment:
Maybe I have missed something: Should the `HueLightDiscoveryService` implement the `SensorStatusListener` interface as well?